code/dqc/anomalychk.q

```diff
@@ -0,0 +1,10 @@
+\d .dqc
+/- Check percentage of anomalies in each of the columns of t, where the columns 
+/- to watch are specified in colslist, and a percentage threshold thres.
+anomalychk:{[t;colslist;thres]
```

Comment:
Logging for this could be like:
```suggestion
anomalychk:{[t;colslist;thres]
  .log.info[`dqe;"checking ",string[t]," for anomalies in columns ",", "sv string(),colslist];
```
Depends if `t` is a symbol or an actual reference to the table. If it can be either then a generic comment about checking for anomalies would be required.

Reply:
iirc it is a symbolic name of table, which makes sense cause we use a `get t` a few lines down